### DISCUSSION TITLE
Access table via REST API

### DISCUSSION THREAD

**Discussion by paraschopra** [AntlerVC/firetable#879] [Support / Q&A] [answered]
Might be a noob question, but I'm looking to acccess table data (and post to it) via  REST API. How do I do that?

**Comment by htuerker** [comment, marked as answer]
Hey @paraschopra, 

I've never used but there's a Restful API for accessing firestore data. https://firebase.google.com/docs/firestore/use-rest-api

But, the recommended way is to build your own back-end endpoints by using a firebase server SDK. You can check the Node.js client library here https://googleapis.dev/nodejs/firestore/latest/
